Makefile

```diff
@@ -155,27 +143,6 @@ clean-target-dir:
 	fi
 .PHONY: clean-target-dir
 
-# Dependencies
-
-.venv/bin/python: Makefile
-	rm -rf .venv
-
-	@# --copies is necessary because OS X make checks the mtime of the symlink
-	@# target (/usr/local/bin/python), which is always much older than the
-	@# Makefile, and then proceeds to unconditionally rebuild the venv.
-	$$RELAY_PYTHON_VERSION -m venv --copies .venv
-	.venv/bin/pip install -U pip wheel
-
-.venv/python-requirements-stamp: requirements-dev.txt
-	.venv/bin/pip install -U -r requirements-dev.txt
-	RELAY_DEBUG=1 .venv/bin/pip install -v --editable py
-	# Bump the mtime of an empty file.
-	# Make will re-run 'pip install' if the mtime on requirements-dev.txt is higher again.
-	touch .venv/python-requirements-stamp
-
-.git/hooks/pre-commit:
```

Comment:
this has been rolled up into `pre-commit install --install-hooks` in `devenv sync`